airflow/providers/google/cloud/hooks/dataprep.py

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+This module contains Google Dataprep hook.
+"""
+import os
+
+import requests
+from tenacity import retry, stop_after_attempt, wait_exponential
+
+TOKEN = os.environ["DATAPREP_TOKEN"]
```

Comment:
importing this file will fail when there no `DATAPREP_TOKEN` in env variables:
```
In [1]: import os

In [2]: os.environ["DATAPREP_TOKEN"]
-----------------------------------------------------------------------
KeyError                              Traceback (most recent call last)
<ipython-input-2-750da8680fbe> in <module>
----> 1 os.environ["DATAPREP_TOKEN"]

~/.virtualenvs/airflow/lib/python3.6/os.py in __getitem__(self, key)
    667         except KeyError:
    668             # raise KeyError with the original key value
--> 669             raise KeyError(key) from None
    670         return self.decodevalue(value)
    671

KeyError: 'DATAPREP_TOKEN'
```

Reply:
This temporary solution. I don't how to resolve it yet.

Reply:
We should add new connection. Like `dataprep_conn_id`. Then the `TestGoogleDataprepHook` should have 
```python
@property
def token(self) -> str:
    connection = self.get_connection(self.dataprep_conn_id)
    try:
        return connection.extra_dejson["token"]
    except KeyError:
        raise AirflowException("Some useful info")
```

This assumes that users should create `dataprep_conn_id` with the following extras: `{"token": USER_DATAPRE_TOKEN}`.

Reply:
Done